### ISSUE TITLE
Camunda Integration shows Invalid Credentials with formsflow.ai docker deployment

### ISSUE THREAD

**Issue by pruthvik9** [AOT-Technologies/forms-flow-ai#978]
## Bug Report
Camunda Integration shows Invalid Credentials with formsflow.ai docker deployment### Please provide information about your setup

- formsflow.ai Version: 5.0.4
- Browser: Google Chrome
- Browser version: Version 106.0.5249.119 (Official Build) (arm64)

### Steps to Reproduce

1. Deploy Keycloak as Individual docker service 
2. Deploy Camunda (bpm) as individual docker service
3. Try to login with the realm credentials created on keycloak with camunda-admin access

### Observed Behaviour
SHOWS INVALID CREDENTIALS 
<img width="1503" alt="Screenshot 2022-10-25 at 17 57 18" src="https://user-images.githubusercontent.com/44010539/197760412-9dd69760-ae94-4ab9-84a4-e50eb36dbbc7.png">

### Expected Behaviour
Realm users on Keycloak should be able login to the camunda UI

**Comment by sumesh-aot** [comment]
@alanraju-aot Can you please take a look ?

**Comment by alanraju-aot** [comment]
@pruthvik9  Can you just confirm whether you are getting error during the initial login through your localhost?
These are the steps you should do while installing bpm through docker-full-deployment
1. install keycloak first
2. bring up all the containers with the command docker-compose up --build -d
3. if its through individual service deployment, bring up keycloak, then bpm, wait for 3 mints for bpm to get integrate with keycloak, then login through keycloak credentials.

**Comment by pruthvik9** [comment]
@alanraju-aot yes this is when I initially log in as localhost (I'm using the server's Private IP). 
Also FYI - I get a prompt to reset the password when I use the default credentials as mentioned [here](https://aot-technologies.github.io/forms-flow-ai-doc/default%20user%20credential.html) 

Also as you suggested let me wait for sometime once camunda containers are up and then try to login with realm credentials

**Comment by alanraju-aot** [comment]
Could you also try quick installation as said in the document. Feel free to report any bugs or error while you install formsflow. Also just try it in your local with the IPv4 address in your system.

**Comment by alanraju-aot** [comment]
@pruthvik9  I think it might be an issue with the IP, I had checked in all ways , tested multiple times. Just try it oncemore, and if the issue still affects you, we can arrange a meeting regarding the issue and lets sort it out.

**Comment by pruthvik9** [comment]
@alanraju-aot just a quick query on keycloak configuration, if it enabled for external authentication (other than localhost or same host) by default?

**Comment by pruthvik9** [comment]
@alanraju-aot Attaching my camunda environment file which is located at - /forms-flow-ai/tree/master/forms-flow-bpm/.env
Please let me know if I am missing anything. 

NOTE : before executing docker compose in this directory, keycloak, forms.io and analytics (redash) components and the uptime was verified. 
[camunda_env.txt](https://github.com/AOT-Technologies/forms-flow-ai/files/9877800/camunda_env.txt)

**Comment by alanraju-aot** [comment]
@pruthvik9 Their is an issue in the env file. After you build keycloak , you must uncomment the keycloak_bpm_client_secret key,
but through quick installation that automatically configures. Can you just try after uncommenting it.

**Comment by pruthvik9** [comment]
@alanraju-aot the issue still persists. Appreciate if we could get on a short call

**Comment by alanraju-aot** [comment]
@pruthvik9 We have traced the issue that you have raised, we will be able to resolve the issue soon, will update the status here. Once the issues are closed, we will let you know.

**Comment by alanraju-aot** [comment]
If you prefer, lets have a call in detail and that can be more effective in understanding the issue and we may catch it sson, please let me know your availability for the call and just add your mail id so I can send you the invitation.

**Comment by pruthvik9** [comment]
pruthvisk4@gmail.com

**Comment by uzrnem** [comment]
@alanraju-aot @pruthv1k did we get solution to this issue, we are also facing same issue while deploying app on server.
cc @sreehari-aot @kiran-aot

**Comment by uzrnem** [comment]
@pruthvik9 Can you just confirm whether you are getting error during the initial login through your localhost? are you able to get rid of this issue?
cc @alanraju-aot

**Comment by sabalalani** [comment]
Hi, Im also facing this issue. Let me know if it has been resolved.

**Comment by sreehari-aot** [comment]
This is a known issue, the temporary fix will be available in the v5.1.0 release
